src/components/Navbar/index.css

```diff
@@ -1,5 +1,5 @@
 .navbar {
-  height: 100vh;
+  height: 100%;
```

Comment:
Please make sure that the navbar has the minimum height of 100vh. For instance, this is how it looks like if there is no content
![image](https://user-images.githubusercontent.com/50754458/229946149-8a7ac5de-311e-4b7c-aadd-ea082b1e137a.png). By adding a `min-height`: 100vh, you will make sure that with or without the content, the copyright text will be at the end of the page